modules/kmm-day1-kernel-module-loading.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: CONCEPT
+[id="kmm-day1-kernel-module-loading_{context}"]
+= Day 1 kernel module loading
+
+Kernel Module Management (KMM) is typically a Day 2 Operator. Kernel modules are loaded only after the complete initialization of a Linux (RHCOS) server. However, in some scenarios the kernel module must be loaded at an earlier stage. Day 1 functionality allows customer to load kernel module during Linux `systemd` initialization stage, using the Machine Config Operator (MCO).
```

Comment:
```suggestion
Kernel Module Management (KMM) is typically a Day 2 Operator. Kernel modules are loaded only after the complete initialization of a Linux (RHCOS) server. However, in some scenarios the kernel module must be loaded at an earlier stage. Day 1 functionality allows the customer to load kernel module during Linux `systemd` initialization stage, using the Machine Config Operator (MCO).
```

Reply:
Sorry, I think there is a missing word here?

Reply:
Thanks. Good catch. Looking at this, I'm going to reword that sentence to _Day 1 functionality allows you to use the Machine Config Operator (MCO) to load kernel modules during the Linux `systemd` initialization stage._